File: CS/CS.NET-VB.NET/ASP.NET 4.0 Book/Source/Chapter 03/VB/HttpHandlerVB/App_Code/CustomHandler.vb
Imports Microsoft.VisualBasic
Namespace CustomHTTP
    Public Class CustomHandler
        Implements IHttpHandler
        Public ReadOnly Property IsReusable() As Boolean Implements System.Web.IHttpHandler.IsReusable
            Get
                Return True
            End Get
        End Property
        Public Sub ProcessRequest(ByVal context As System.Web.HttpContext) Implements System.Web.IHttpHandler.ProcessRequest
            Dim Response As HttpResponse = context.Response
            Response.Write("<html><body><H2>Sample HTTP Handler</H2></body></html>")
        End Sub
    End Class
End Namespace

File: CS/CS.NET-VB.NET/ASP.NET 4.0 Book/Source/Chapter 03/CS/HttpHandlerCS/App_Code/CustomHandler.cs
using System;
using System.Data;
using System.Configuration;
using System.Linq;
using System.Web;
using System.Web.Security;
using System.Web.UI;
using System.Web.UI.HtmlControls;
using System.Web.UI.WebControls;
using System.Web.UI.WebControls.WebParts;
using System.Xml.Linq;


/// <summary>
/// Summary description for CustomHandler
/// </summary>
namespace CustomHTTP
{
    public class CustomHandler : IHttpHandler
{
    public bool IsReusable
    {
        get
        {
            return true;
        }
    }
    public void ProcessRequest(System.Web.HttpContext context)
    {
        HttpResponse Response = context.Response;
        Response.Write("<html><body><H2>Sample HTTP Handler</H2></body></html>");
    }

}

}
